lib/api/protocol/waitUntil.js

```diff
@@ -61,6 +61,10 @@ module.exports = class WaitUntil extends ProtocolAction {
     return true;
   }
 
+  static get resolveParentNodeWithResult() {
+    return false;
+  }
+
```

Comment:
Or probably we should make it negative so that it's easier to handle later.

Right now, if we don't specify `resolveParentNodeWithResult` in a file, it will evaluate to `undefined` which is also a `falsy` value, so it'll be difficult to distinguish that from when we've specified a `false` value.

On the other hand, if we make the getter negative by changing the name to `avoidPrematureParentNodeResolution()`, we can return `true` from it when we want and in all the other cases, it will return a `falsy` value (`undefined`).